netbox/resource_netbox_device_interface_test.go

```diff
@@ -0,0 +1,273 @@
+package netbox
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/fbreckle/go-netbox/netbox/client"
+	"github.com/fbreckle/go-netbox/netbox/client/dcim"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	log "github.com/sirupsen/logrus"
+)
+
+func testAccNetboxDeviceInterfaceFullDependencies(testName string) string {
+	return fmt.Sprintf(`
+
+resource "netbox_tag" "test" {
+  name = "%[1]s"
+}
+
+resource "netbox_site" "test" {
+  name = "%[1]s"
+  status = "active"
+}
+
+resource "netbox_device_role" "test" {
+  name = "%[1]s"
+  color_hex = "123456"
+}
+
+resource "netbox_manufacturer" "test" {
+	name = "%[1]s"
```

Comment:
Inconsistent formatting in the leading whitespace

Reply:
thank you, has been resolved